dashboards/defaults.libsonnet

```diff
@@ -7,6 +7,22 @@
     [filename]: grafanaDashboards[filename] {
       uid: std.md5(filename),
       timezone: '',
+
+      // Modify tooltip to only show a single value
+      rows: [
+        row {
+          panels: [
+            panel {
+              tooltip+: {
```

Comment:
How about instead of just setting this for any dashboard doing something along the lines of this here?

```jsonnet
[if filename in ['dashboarda.json', 'dashboardb.json'] then "tooltip"]
```